lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -170,31 +170,24 @@ export default function ContentSelector({
   };
 
   // file.id is a URL with a `blackboard://`, `d2l://` or `moodle://` prefix.
-
-  const selectFileAsURL = (file: File) => selectURL(file.id);
-  const selectPageAsURL = (page: File, lms: string) => {
-    const name = `${lms} page: ${page.display_name}`;
-    selectURL(page.id, name);
+  const selectContent = (file: File, preffix: string) => {
+    const name = `${preffix}: ${file.display_name}`;
+    selectURL(file.id, name);
   };
 
-  const selectCanvasFile = (file: File) => {
-    selectURL(file.id);
-  };
+  const selectCanvasPage = (page: File) => selectContent(page, 'Canvas page');
```

Comment:
Do we need/want all this names? Could the prefix be configured/passed from the callers below?

Reply:
Yes, we could definitely do that.

This is a side effect of the refactor I did some days ago. Previously, these were all different functions with "duplicate" code that I consolidated in the `selectURL`.

After those changes, your point is very valid. I can put together a PR refactoring that.